lib/screens/login_screen/widgets/custom_login_widget.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:tsec_app/provider/theme_provider.dart';
+import 'package:tsec_app/utils/themes.dart';
+
+class LoginWidget extends StatefulWidget {
+  @override
+  _LoginWidgetState createState() => _LoginWidgetState();
+}
+
+class _LoginWidgetState extends State<LoginWidget> {
+  @override
+  Widget build(BuildContext context) {
+    var brightness = Theme.of(context).brightness;
```

Comment:
can use final here instead of var, haven't you got a warning from linter?

Reply:
No Warning from linter but will fix it.

Reply:
Need to add warning for this in linter